WeakAurasArchive/WeakAurasArchive.lua

```diff
@@ -0,0 +1,12 @@
+-- all this is is a frame that ensures that the SV is a table type
+local addonName = ...
+local loader = CreateFrame("FRAME")
+loader:RegisterEvent("ADDON_LOADED")
+loader:SetScript("OnEvent", function(self, _, addon)
+  if addon == addonName then
+    if type(WeakAurasArchive) ~= "table" then
+      WeakAurasArchive = {}
+    end
```

Comment:
Currently, that would not be useful. If it's time for an archive cleanup then it would have been noticed at ADDON_LOADED of the main WeakAuras addon (currently tuned to once per day, though we could easily tune that frequency.